doc/eventhandler/usernotification.rst

```diff
@@ -11,34 +11,103 @@ administrators in case of any event.
 Possible Actions
 ~~~~~~~~~~~~~~~~
 
+.. _sendmail_action:
+
 sendmail
 ........
 
-The *sendmail* action sends an email to the tokenowner user. The email is
-sent, if an administrator managed the users token.
+The *sendmail* action sends an email to the specified email address each time
+the event handler is triggered.
 
 **emailconfig**
 
   * *required* Option
-  * The email is sent via this :ref:`smtpserver`.
+
+The email is sent via this :ref:`smtpserver`.
+
+**To**
+
+  * *required* Option
+
+This specifies to which type of user the notification should be sent.
+Possible recipient types are:
+
+  * token owner,
+  * logged in user,
+  * admin realm,
+  * internal admin,
+  * email address.
+
+Depending on the recipient type you can enter additional information. The
+recipient type *email* takes a comma separated list of email addresses.
+
+**reply_to**
+
+Adds the specified ``Reply-To`` header to the email.
 
 **subject**
 
-  * optional
+The subject can take the same tags as the body, except for the ``{googleurl_img}``.
+
+**mimetype**
+
+Possible mime types are:
 
-The subject line of the mail that is sent.
+  * plain (default)
+  * html
+
+You can choose if the email should be sent as plain text or HTML. If the
+email is sent as HTML, you can do the following::
+
+   <a href={googleurl_value}>Your new token</a>
+
+Which will create a clickable link. Clicked on the smartphone, the token will
+be imported to the smartphone app.
+
+You can also do this::
+
+  <img src={googleurl_img}>
+
+This will add the QR Code as an inline data image into the HTML email.
+
+.. warning:: The KEY URI and the QR Code contain the secret OTP key in plain
+   text. Everyone who receives this data has a detailed copy of this token.
+   Thus we very much recommend to **never** send these data in an unencrypted
+   email!
+
+**attach_qrcode**
+
+Instead of sending the QR-Code as an inline data image (which is not supported
+by some email clients / webmailer), enabling this option sends the email as a
```

Comment:
Should we add something "... like Outlook", since people usually have problems with that?

Reply:
will do

Reply:
done in 7b39aa7e